layout.tt

```diff
@@ -117,6 +117,18 @@ BLOCK navigationLink %]
       </div>
       <hr />
       <div class="lower">
+        <div>
+          <span>
+            The NixOS Foundation aims to promote participation without
+            regard to gender, sexual orientation, disability, ethnicity,
+            age, or similar personal characteristics. We want to strive
+            to create and foster community by providing an intentionally
+            welcoming and safe environment where all feel valued and cared
+            for, and where all are given opportunity to participate
+            meaningfully. The Foundation will work with the community in
+            service of this goal.
+          </span>
+        </div>
```

Comment:
Unclassified `<div>` are forbidden. Use a `<section>`, and add a class e.g. `footer-statement`.

The `<span>` serves no purpose here. It should be removed. Though a `<p>` would be needed as this is more akin to *copy text* than some form of *functional text*. So rather than outright removing the `<span>` a `<p>` should be used.